sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -670,9 +675,22 @@ def __init__(
             **ops_kwargs,  # type: ignore[arg-type]
         )
 
+        self._serverless_endpoints = ServerlessEndpointOperations(
+            self._operation_scope,
+            self._operation_config,
+            self._service_client_01_2024_preview,
+            self._operation_container,
+        )
+        self._marketplace_subscriptions = MarketplaceSubscriptionOperations(
+            self._operation_scope,
+            self._operation_config,
+            self._service_client_01_2024_preview,
```

Comment:
Quick check: These are fine with the 01 API restclient?

Reply:
yup, they are